src/meta/src/dashboard/prometheus.rs

```diff
@@ -124,3 +124,44 @@ pub async fn list_prometheus_cluster<S: MetaStore>(
         Err(err(anyhow!("Prometheus endpoint is not set")))
     }
 }
+
+#[derive(Serialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct ActorBackPressure {
+    output_buffer_blocking_duration: Vec<PrometheusVector>,
+}
+pub async fn list_prometheus_actor_back_pressure<S: MetaStore>(
+    Extension(srv): Extension<Service<S>>,
+) -> Result<Json<ActorBackPressure>> {
+    if let Some(ref client) = srv.prometheus_client {
+        let now = SystemTime::now();
+        let back_pressure_query = "rate(stream_actor_output_buffer_blocking_duration_ns{job=~\"compute\"}[60s]) / 1000000000";
```

Comment:
```suggestion
        let back_pressure_query = "rate(stream_actor_output_buffer_blocking_duration_ns{job=~\"compute\"}[15s]) / 1000000000";
```

I think it need to aligned with `query_range`'s interval (line 149)?

Reply:
IIUC, the `step` in `query_range` doesn't seem to be related to `rate_interval`.